docs/source/bigchaindb-cli.md

```diff
@@ -20,3 +21,65 @@ There's another command named `bigchaindb-benchmark`. It's used to run benchmark
 $ bigchaindb-benchmark -h
 $ bigchaindb-benchmark load -h
 ```
+
+Note that you can always start `bigchaindb` using a different config file using the `-c` option.
+For more information check the help with `bigchaindb -h`.
+
+
+# Precedence in reading configuration values
+
+Note that there is a precedence in reading configuration values:
+ - (**not yet**) command line;
```

Comment:
Maybe remove this line (about config values set from the command line) for now, until we actually implement that (if ever).